src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -353,17 +856,42 @@
 		"sourcePath" : "log.sagebase.org/certifiedUserPassingRecords/",
 		"tableDescriptor": {
 			"name": "certifiedquizrecords",
-			"columns": {
-				"response_id": "bigint",
-				"user_id": "bigint",
-				"passed": "boolean",
-				"passed_on": "timestamp",
-				"stack": "string",
-				"instance": "string"
+			"description" : "This table contain records of the certification quiz taken by a Synapse user.",
+			"columns":[{
+				"name": "response_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier for the instance of a quiz taken by user."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The identifier of the user who took the quiz."
+			},
+			{
+				"name": "passed",
+				"type" : "boolean",
+				"comment" : "If true, user passed the quiz."
 			},
-			"partitionKeys": {
-				"record_date": "date"
+			{
+				"name": "passed_on",
+				"type" : "timestamp",
+				"comment" : "The date on which the user took the quiz."
 			},
+			{
+				"name": "stack",
+				"type" : "string",
+				"comment" : "The stack (prod, dev) on which the quiz record was processed."
+			},
+			{
+				"name": "instance",
+				"type" : "string",
+				"comment" : "The unique identifier for the instance of a quiz taken by user."
```

Comment:
Is this essentially the version of the quiz?

Reply:
Wrong description it is  "The version of the stack that processed the quiz record."